### PR TITLE
Rework reviews

### LINKED ISSUE #1142: Redesign Schedule Interview/etc

### PR DESCRIPTION
**Pull request by haiodo**
Closes #1142

Signed-off-by: Andrey Sobolev [haiodo@gmail.com](mailto:haiodo@gmail.com)

### CODE REVIEW

File: plugins/calendar-resources/src/plugin.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import calendar, { calendarId } from '@anticrm/calendar'
+import { mergeIds } from '@anticrm/platform'
+
+export default mergeIds(calendarId, calendar, {
```
Comment (BykhovDenis) [inline plugins/calendar-resources/src/plugin.ts:19]: Why not just {}?

File: plugins/recruit-resources/src/components/review/ReviewCategoryPresenter.svelte [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  import type { ReviewCategory } from '@anticrm/recruit'
+  import { Icon } from '@anticrm/ui'
+  import { showPanel } from '@anticrm/ui/src/panelup'
+  import recruit from '../../plugin'
+
+  export let value: ReviewCategory
+  export let inline: boolean = false
+
+  function show () {
+    // showPanel(recruit.component.EditVacancy, value._id, value._class, 'right')
```
Comment (BykhovDenis) [inline plugins/recruit-resources/src/components/review/ReviewCategoryPresenter.svelte:26]: Do you fix it?

File: packages/query/src/index.ts [hunk 3]
```diff
@@ -624,7 +624,8 @@ export class LiveQuery extends TxProcessor implements Client {
             if (q.options !== undefined) {
               const lookup = (q.options.lookup as any)?.[pkey]
               if (lookup !== undefined) {
-                ;(updatedDoc.$lookup as any)[pkey].push(await this.client.findOne(lookup, { _id: (pops as any)[pkey] as Ref<Doc> }))
+                const pp = updatedDoc.$lookup as any
+                pp[pkey] = (pp[pkey] ?? []).concat(await this.client.findOne(lookup, { _id: (pops as any)[pkey] as Ref<Doc> }))
```
Comment (BykhovDenis) [inline packages/query/src/index.ts]: It will conflict with #1161

**Review by BykhovDenis — APPROVED** [review]

File: packages/query/src/index.ts (hunk 3 above)
Comment (haiodo) [reply packages/query/src/index.ts]: Resolved

File: plugins/recruit-resources/src/components/review/ReviewCategoryPresenter.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/recruit-resources/src/components/review/ReviewCategoryPresenter.svelte:26]: Will fix in next PR

File: plugins/calendar-resources/src/plugin.ts (hunk 1 above)
Comment (haiodo) [reply plugins/calendar-resources/src/plugin.ts:19]: Plan to fill it in next PRs.

**Pull request merged**